[Bug] rat check fails

### Before Creating the Bug Report

- [X] I found a bug, not just asking a question, which should be created in [GitHub Discussions](https://github.com/apache/rocketmq/discussions).

- [X] I have searched the [GitHub Issues](https://github.com/apache/rocketmq/issues) and [GitHub Discussions](https://github.com/apache/rocketmq/discussions)  of this repository and believe that this is not a duplicate.

- [X] I have confirmed that this bug belongs to the current repository, not other repositories of RocketMQ.


### Runtime platform environment

NA

### RocketMQ version

5.2.x

### JDK Version

NA

### Describe the Bug

Rat-check fails in `rocketmq-common`.

### Steps to Reproduce

Run `mvn apache-rat:check`.

### What Did You Expect to See?

The command above exits successfully.

### What Did You See Instead?

```
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.12:check (default-cli) on project rocketmq-common: Too many files with unapproved license
```

### Additional Context

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -314,13 +314,14 @@
                         <exclude>src/test/resources/**</exclude>
                         <exclude>src/test/resources/certs/*</exclude>
                         <exclude>src/test/**/*.log</exclude>
-                        <exclude>src/test/resources/META-INF/service/*</exclude>
-                        <exclude>src/main/resources/META-INF/service/*</exclude>
+                        <exclude>src/test/resources/META-INF/services/*</exclude>
+                        <exclude>src/main/resources/META-INF/services/*</exclude>
                         <exclude>*/target/**</exclude>
                         <exclude>*/*.iml</exclude>
                         <exclude>docs/**</exclude>
                         <exclude>localbin/**</exclude>
                         <exclude>conf/rmq-proxy.json</exclude>
+                        <exclude>.bazelversion</exclude>
                     </excludes>
                 </configuration>
             </plugin>

